xmodule/library_tools.py

```diff
@@ -128,12 +128,15 @@ def trigger_library_sync(self, dest_block: LibraryContentBlock, library_version:
             if library_version:
                 raise ObjectDoesNotExist(f"Version {library_version} of library {library_key} not found.")
             raise ObjectDoesNotExist(f"Library {library_key} not found.")
-        library_tasks.sync_from_library.delay(
+
+        # TODO: This task is synchronous until we can figure out race conditions with import
+        # These race conditions lead to failed imports of library content from course import.
+        # See: TNL-TNL-11339, https://github.com/openedx/edx-platform/issues/34029 for more info.
+        library_tasks.sync_from_library.apply(
```

Comment:
https://docs.celeryq.dev/en/latest/reference/celery.app.task.html is how you know apply is the method to use.